ui/plugin-lifecycle/src/public/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "urn": "vmware:vcloud:plugin:seed",
```

Comment:
update urn and route to be more appropriate for a published plugin

Reply:
Fixed in initial commit.